docs/framework/wpf/getting-started/walkthrough-my-first-wpf-desktop-application.md

```diff
@@ -54,17 +54,19 @@ The first step is to create the application infrastructure, which includes an ap
 
       The **Create a new project** dialog opens.
 
-      ![Create a new project dialog](./media/gettingstartedfigure0a.png)
+      ![Create a new project dialog](./media/walkthrough-my-first-wpf-desktop-application/create-new-project-dialog.png)
 
    2. In the **Language** dropdown, select either **C#** or **Visual Basic**.
 
+      ![New Project dialog with WPF app selected](./media/walkthrough-my-first-wpf-desktop-application/new-project-dialog-box.png)
```

Comment:
this seems it would belong under step 3 instead with a .NET Framework 4.8 target since it's our latest version.

A possibility here would also be to move the previous version down to step 2 with the language dropdown location highlighted in red.